### PR TITLE
Added copy-locale for snippet UI

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [x] test coverage
- [ ] gather feedback for my changes

__Related PR:__

* https://github.com/massiveart/husky/pull/504

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes part of #1331
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+define(function() {
+
+    'use strict';
+
+    var templates = {
+            copyLocales: function(item) {
+                var template = [
+                    '<div class="copy-locales-overlay-content">',
+                    '   <label>',
+                    this.sandbox.translate('content.contents.settings.copy-locales.copy-from'),
```
Comment (marcelmoosbrugger) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js]: @wachterjohannes Wouldn't it be better to make the templates as strings not functions and pass the needed text via the underscore-templating method?

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                                        type: 'cancel',
+                                        align: 'right'
+                                    },
+                                    {
+                                        type: 'ok',
+                                        text: this.sandbox.translate('content.contents.settings.copy-locales.ok'),
+                                        align: 'left'
+                                    }
+                                ],
+                                okCallback: function() {
```
Comment (marcelmoosbrugger) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js]: @wachterjohannes this callback is pretty long. please make it an own method

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+                    }
+                }.bind(this));
+        };
+
+    return {
+        copyLocale: function(id, src, dest, successCallback, errorCallback) {
+            copyLocale.call(this, id, src, dest, successCallback, errorCallback);
+        },
+
+        startCopyLocalesOverlay: function() {
```
Comment (marcelmoosbrugger) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js]: this method is huge. maybe you can split it into a few smaller parts.

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js [hunk 4]
```diff
@@ -0,0 +1,203 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(function() {
```
Comment (marcelmoosbrugger) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js]: @wachterjohannes some comments over the components-methods would be nice

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/open-ghost-overlay/main.js [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(function() {
+
+    'use strict';
+
+    var templates = {
+            openGhost: function() {
```
Comment (marcelmoosbrugger) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/open-ghost-overlay/main.js]: @wachterjohannes see above. (templates as functions)

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js (hunk 1 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js]: its hard to refactor and its moved from another component ...

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js (hunk 2 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js]: fixed

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js (hunk 3 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js]: fixed

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js (hunk 4 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js]: fixed

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/open-ghost-overlay/main.js (hunk 5 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/open-ghost-overlay/main.js]: fixed

**Pull request merged**